build.gradle

```diff
@@ -32,25 +32,28 @@ configurations {
 
 // The repackaging rules are defined in the "shadowJar" task below.
 dependencies {
-    compile "com.android.tools:r8:1.0.37"
+    compile "com.android.tools:r8:1.4.93"
```

Comment:
1.5 will be released this week with AS 3.5 stable so you might want to wait a day or two here.

Reply:
AS 3.5 is out now, I checked r8 still no new update. ok, I'll wait and update later. For the time being, I'll just use my own executableJar

Reply:
You can use R8 1.5.64 (bundled on AS 3.5.0) with following dependency.

```
diff --git a/build.gradle b/build.gradle
index 58411da..79e3684 100755
--- a/build.gradle
+++ b/build.gradle
@@ -22,6 +22,9 @@ repositories {
     mavenLocal()
     jcenter()
     google()
+    maven {
+        url "http://storage.googleapis.com/r8-releases/raw"
+    }
 }
 
 configurations {
@@ -32,7 +35,7 @@ configurations {
 
 // The repackaging rules are defined in the "shadowJar" task below.
 dependencies {
-    compile "com.android.tools:r8:1.0.37"
+    compile "com.android.tools:r8:1.5.64"
     compile "com.android.tools.build:apkzlib:3.4.0-beta01"
     compile "com.android.tools.ddms:ddmlib:26.2.0"
 
```

Reply:
> You can use R8 1.5.64 (bundled on AS 3.5.0) with following dependency.
> 
> ```
> diff --git a/build.gradle b/build.gradle
> index 58411da..79e3684 100755
> --- a/build.gradle
> +++ b/build.gradle
> @@ -22,6 +22,9 @@ repositories {
>      mavenLocal()
>      jcenter()
>      google()
> +    maven {
> +        url "http://storage.googleapis.com/r8-releases/raw"
> +    }
>  }
>  
>  configurations {
> @@ -32,7 +35,7 @@ configurations {
>  
>  // The repackaging rules are defined in the "shadowJar" task below.
>  dependencies {
> -    compile "com.android.tools:r8:1.0.37"
> +    compile "com.android.tools:r8:1.5.64"
>      compile "com.android.tools.build:apkzlib:3.4.0-beta01"
>      compile "com.android.tools.ddms:ddmlib:26.2.0"
>  
> ```

Could not find com.android.tools:r8:1.5.64

Reply:
It's 1.5.68, and it's out now